SofaKernel/modules/SofaComponentBase/SofaComponentBase_test/MakeDataAliasComponent_test.cpp

```diff
@@ -79,7 +79,7 @@ void perTestInit()
     }
 
     if(defaultHandler==nullptr)
-        defaultHandler=new ConsoleMessageHandler(new RichConsoleStyleMessageFormatter) ;
+        defaultHandler=new ConsoleMessageHandler(&RichConsoleStyleMessageFormatter::getInstance()) ;
```

Comment:
Not really related with the PR but these codes are not needed anymore if the test is based on BaseTest.